Documentation/cluster-crd.md

```diff
@@ -295,3 +295,47 @@ spec:
           cpu: "2"
           memory: "4096Mi"
 ```
+
+## RBAC for PodSecurityPolicies
```

Comment:
As this is not a Cluster CRD option, I think this should only be put in the `kubernetes.md` so everything required is in the same file.